### DISCUSSION TITLE
Is it possible to position the command / addressbar at the bottom?

### DISCUSSION THREAD

**Discussion by tankred** [Jelmerro/Vieb#465] [Q&A] [answered]
Is it possible to position the command / addressbar at the bottom? Maybe in .viebrc

**Comment by Jelmerro** [comment, marked as answer]
See colorschemes, and the example colorscheme/theme named flipped. Set it with `:colorscheme flipped` or see the help with `:h colorscheme`.

**Reply by tankred** [reply]
Excellent this works fine. Thank you.
